AY2324S1-CS2103T-W15-3/tp
# Add tests for view and other member related utils

Addresses #97 & #98 

also addresses:
- properly implement ClearCommandTest, and TypicalMembers, then remove TypicalPersons (look into a way to create addressBook with both applicants and members for testing?)
- properly implement AddressBookTest
- properly implement ModelManagerTest
- add ApplicantTest and MemberTest, then remove PersonTest
- properly implement UniquePersonListTest
- add tests for new fields, e.g. TelegramTest
- add JsonAdaptedMemberTest, then remove JsonAdaptedPersonTest
- properly implement JsonAddressBookStorageTest
- properly implement StorageManagerTest after getTypicalAddressBookWithMember() (TypicalMembers)
- add MemberUtil and MemberBuilder, then remove PersonUtil and PersonBuilder

@Saezenn plss help me check esp the assertions in StorageManagerTest.java at lines 47 & 63, and also in JsonAddressBookStorageTest. also can help close the 2 issues above if you merge thanks!
